libs/core/execution/include/hpx/execution/algorithms/schedule_from.hpp

```diff
@@ -292,46 +291,38 @@ namespace hpx { namespace execution { namespace experimental {
 
             template <typename Receiver>
             friend operation_state<Receiver> tag_invoke(
-                connect_t, on_sender&& s, Receiver&& receiver)
+                connect_t, schedule_from_sender&& s, Receiver&& receiver)
             {
                 return {HPX_MOVE(s.predecessor_sender), HPX_MOVE(s.scheduler),
                     HPX_FORWARD(Receiver, receiver)};
             }
 
             template <typename Receiver>
             friend operation_state<Receiver> tag_invoke(
-                connect_t, on_sender& s, Receiver&& receiver)
+                connect_t, schedule_from_sender& s, Receiver&& receiver)
             {
                 return {s.predecessor_sender, s.scheduler,
                     HPX_FORWARD(Receiver, receiver)};
             }
         };
     }    // namespace detail
 
-    inline constexpr struct on_t final
-      : hpx::functional::detail::tag_fallback<on_t>
+    HPX_HOST_DEVICE_INLINE_CONSTEXPR_VARIABLE struct schedule_from_t final
```

Comment:
`nvcc` was complaining about this not being a device variable, which is why I've changed this to be `HPX_HOST_DEVICE_INLINE_CONSTEXPR_VARIABLE`. However, it's not clear to me what triggers it. Other CPOs are fine as plain `inline constexpr` variables :/ In principle they could all probably be host-device variables, but I'm changing only this one in this PR as a minimal change to make this compile with `nvcc`.